playwright.config.ts

```diff
@@ -1,17 +1,19 @@
 import { defineConfig, devices } from "@playwright/test";
 
+const headless = process.env.HEAD !== 'true';
+
 module.exports = defineConfig({
   testDir: "./playwright_tests/E2E",
   /* Run tests in files in parallel */
   fullyParallel: true,
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
   /* Retry on CI only */
-  retries: 4, // Set the number of retries for all projects
+  retries: 3, // Set the number of retries for all projects
 
-  timeout: 10 * 60 * 1000,  // 10 minutes
+  timeout: 3 * 60 * 1000,  // 10 minutes
```

Comment:
Please fix the comment to say 3 minutes - actually remove the comment it's obvious

Reply:
done